test/test.js

```diff
@@ -7389,6 +7389,65 @@ exports['wrapCallback - default mapper discards all but first arg'] = function (
     });
 };
 
+exports.wrapAsync = function (test) {
```

Comment:
Yup. I just copied the wrapCallback tests. It's good to know that you can group them like this and pass in the expected number of assertions.